use-cases/src/Plutus.hs

```diff
@@ -0,0 +1,138 @@
+{-# LANGUAGE EmptyDataDecls             #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+-- This is a mock of (parts of) the Plutus API
+module Plutus (
+
+  Address, PubKey, KeyPair, Value, standardTxFee, pubKeyAddress,
+
+  T, Tx(..), TxOut(..), TxOutRef(..), Witness, PlutusTx, Validator,
+  submitTransaction, assert, lookupMyPubKey, lookupMyKeyPair, createPayment, txInSign,
+  Range(..),
+  EventTrigger(..)
+) where
+
+import           Language.Haskell.TH        hiding (Range)
+import           Language.Haskell.TH.Syntax hiding (Range)
+{-# ANN module "HLint: ignore Avoid restricted function" #-}
+
+-- |Cardano address
+--
+type Address = String
+
+-- |Ada value
+--
+newtype Value = Value Integer
+              deriving (Eq, Ord, Num, Show, Read)
+
+-- |Public key
+--
+data PubKey
+
+instance Lift PubKey where
+  lift = undefined
+
+-- |Public key pair (no lift instance, because we never ought to put it into a transaction)
+--
+data KeyPair
+
+pubKeyAddress :: KeyPair -> Address
+pubKeyAddress = undefined
+
+data TxState
+
+-- |Transaction monad for coordination layer computations; provides access to the blockchain
+--
+newtype T a = T (TxState -> (TxState, a))

```

Comment:
eh, just use `State`

Reply:
Personally, I'm not a fan of one-letter names. Maybe at least `Tx` since it's `TxState` inside?

Reply:
Ah, I see, `Tx` is used later. `TxM`?